torchft/manager.py

```diff
@@ -191,7 +190,7 @@ def __init__(
 
             if replica_id is None:
                 replica_id = ""
-            replica_id = replica_id + str(uuid.uuid4())
```

Comment:
Wasn't sure why uuid was added to the replica_id, is this needed?

Reply:
ye, I was wondering the same thing. TorchTitan has to keep the original `replicate_id` because this change every time due to `uuid4`.

Reply:
We do this as a simplification for the quorum algorithm -- if a worker restarts quickly with the same name we may not detect that the quorum has changed and thus no-reconfiguration of the PGs will occur. Adding a random UUID to the worker means that we will always detect processes restart and thus correctly trigger a reconfiguration

@H-Huang can you add this back and if it's convenient also add a comment explaining why this is the case? Also it would be nice to add a `:` to it so `replica_01234-1234` shows up as `replica_0:1234-1234` . I've been meaning to fix this but haven't had a chance